### PR TITLE
Add @kdubb to quarkus-vault

### PR DESCRIPTION
**Pull request by gastaldi**
As requested in https://github.com/quarkusio/quarkus/issues/16870#issuecomment-987271126

### CODE REVIEW

**Review by gsmet — CHANGES_REQUESTED** [review]
Let's wait for the final approval of the move.

**Review by vsevel — APPROVED** [review]

**Review by gsmet — APPROVED** [review]

**Pull request merged**
